seqr/views/utils/airtable_utils.py

```diff
@@ -105,49 +104,3 @@ def _populate_records(self, record_type, records, offset=None):
 
         if response_json.get('offset'):
             self._populate_records(record_type, records, offset=response_json['offset'])
-
-
-def _get_airtable_samples_for_id_field(sample_ids, id_field, fields, session):
```

Comment:
these helper functions were only used in one place and needed to be edited. In general, for utilities that are only used once it is better to keep them as private methods in the file where they are used so its easier to know what is and is not safe to change